recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+if [[ "${target_platform}" == "osx-64" ]]; then
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake . -B ./build -G Ninja ${CMAKE_ARGS} -DSEAL_BUILD_DEPS=OFF -DSEAL_BUILD_TESTS=ON
```

Comment:
Applicable to win and unix - 
`-DBUILD_SHARED_LIBS=ON`? (off by default)

Reply:
Windows only supports static, see . So for consistency, I use static everywhere.

Reply:
For now yes. If we ever need to link against both the shared and static, then we could enable the shared library I guess.